src/loadgenerator/Dockerfile

```diff
@@ -12,16 +12,17 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-FROM python:3-slim as base
+FROM python:3.7.2-slim as base
```

Comment:
refrain from using specific versions like 3.7.2.
if there's a security bug tomorrow, 3.7.2 will potentially be deleted.

Reply:
Done